action/protocol/execution/evm/evmstatedbadapter.go

```diff
@@ -48,6 +48,8 @@ type (
 		err                        error
 		blockHeight                uint64
 		executionHash              hash.Hash256
+		lastAddBalanceAddr         string
```

Comment:
getStateObject(addr common.Address) in geth(https://github.com/ethereum/go-ethereum/blob/ebf9e11af2ff701d0961623e817d37b421b96802/core/state/statedb.go#L540) seems more reasonable than two new variables under StateDBAdapter for me

Reply:
can you elaborate? we cannot get the beneficiary's address in StateDBAdapter, so capture it in `AddBalance` every time. also we need the transfer amount, not the beneficiary's entire balance